scrapenews/tests/unit/test_pipelines.py

```diff
@@ -0,0 +1,12 @@
+from unittest import TestCase
+
+from pipelines import AlephPipeline
+
+
+class TestPipeline(TestCase):
```

Comment:
best to remove this for now - it doesn't test anything right now so I'd rather not have the illusion that core functionality is tested.